index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+<main id="main-doc">
+  <section class="main-section" id="Section_1">
+    <header>
+      <p>Section1</p>
+      <p>About <code>this</code>  section</p>
+      <li>Javascript</li>
+    </header>
+  </section>
+  <section class="main-section" id="Section_2">
+    <header>
+      <p>Section2</p>
+      <p>About <code>this</code>  section</p>
+      <li>Java</li>
+    </header>
+  </section>
+  <section class="main-section" id="Section_3">
+    <header>
+      <p>Section3</p>
+      <p>About <code>this</code> section</p>
+      <li>Python</li>
+    </header>
+  </section>
+  <section class="main-section" id="Section_4">
+    <header>
+      <p>Section4</p>
+      <p>About <code>this</code> section</p>
+      <li>Matlab</li>
+    </header>
+  </section>
+  <section class="main-section" id="Section_5">
+    <header>
+      <p>Section5</p>
+      <p>About <code>this</code> section</p>
+      <li>R</li>
+    </header>
+    
+  </section>
+
+</main>
+<nav id="navbar">
+<header>
+  <h1>Diffrent programming language</h1>
+</header>
+<a href = "#" class = "nav-link">Section1</a>
+<a href = "#" class = "nav-link">Section2</a>
+<a href = "#" class = "nav-link">Section3</a>
+<a href = "#" class = "nav-link">Section4</a>
+<a href = "#" class = "nav-link">Section5</a>
```

Comment:
I would suggest adding the links in a container and make it easier to design them.